pkg/chunk/cached_store.go

```diff
@@ -289,10 +288,20 @@ func (c *rChunk) Remove() error {
 		delete(c.store.pendingKeys, key)
 		c.store.pendingMutex.Unlock()
 		c.store.bcache.remove(key)
+
+	}
+
+	if c.store.conf.MaxDeletes == 0 {
+		return nil
```

Comment:
nil means the chunk is removed from object store, then slice will be remove from meta engine, and chunk will be leaked.

We should return EBUSY here, then no error message in deleteSlice()

Reply:
[67a5258](https://github.com/juicedata/juicefs/pull/1611/commits/67a52581a264edb067b4596c97e22909c35e5648)
Not using EBUSY here since that's not accurate enough.